pom.xml

```diff
@@ -37,6 +37,13 @@
     <module>build</module>
     <module>features</module>
   </modules>
+  <repositories>
+	  <repository>
+		  <id>bnd-snapshot</id>
+		  <layout>p2</layout>
+		  <url>https://bndtools.jfrog.io/bndtools/update-snapshot</url>
+	  </repository>
+  </repositories>
```

Comment:
This is only temporary until bnd-lib 7.1 is released and it is included in the sdk-prereq-target.
Has to be reverted before this can be merged.